### PR TITLE
Fix EZP-25014: Impossible to update a policy with no limitations

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-25014

### CODE REVIEW

**Comment by lolautruche** [comment]
Review ping @andrerom @bdunogier @pspanja @glye

**Comment by lolautruche** [comment]
I think this should be backported to 5.4 and 5.3

**Comment by andrerom** [comment]
+1 *(and also for backport)*

File: eZ/Publish/API/Repository/Tests/RoleServiceTest.php [hunk 1]
```diff
@@ -1366,6 +1366,52 @@ public function testNewPolicyUpdateStruct()
...
+
+        // Search for the new policy instance
+        $policy = null;
+        foreach ($role->getPolicies() as $policy) {
+            if ($policy->module === 'foo' && $policy->function === 'bar') {
+                break;
+            }
+        }
+
+        // Create an update struct and set a modified limitation
```
Comment (glye) [inline eZ/Publish/API/Repository/Tests/RoleServiceTest.php]: "set a modified limitation" <- seems you're not doing that

**Comment by pspanja** [comment]
+1

**Comment by glye** [comment]
+1 apart from comment nitpick

**Comment by lolautruche** [comment]
Manually merged: 570e25c198aa6199a237f7aecfbe67d6e9e80237

**Comment by lolautruche** [comment]
@glye I updated the comment in the final commit.

**Pull request closed without merging**